dev-utils/ci-tav-slow-jobs.sh

```diff
@@ -19,6 +19,12 @@ fi
 set -o errexit
 set -o pipefail
 
+# Make sure we have json package
+jsonPath=$(which json)
+if [ "$jsonPath" == "" ]; then
+    npm i -g json
```

Comment:
I *really* don't like a script making global changes to my system as a side-effect. I would prefer we just cope with the occasional failure, or how about we just add `json.js` to our dev-utils/ area. It is a single JS file?